templates/type.mustache

```diff
@@ -3,10 +3,8 @@
 {{=<% %>=}}
 <%#isRef%><%target%><%/isRef%><%!
 %><%#isAtomic%><%&tsType%><%/isAtomic%><%!
-%><%#isObject%>{<%#properties%>
-'<%name%>'<%^isRequired%>?<%/isRequired%>: <%>type%><%/properties%>
-}<%/isObject%><%!
+%><%#isObject%>{<%#properties%>'<%name%>'<%^isRequired%>?<%/isRequired%>: <%>type%><%/properties%>}<%/isObject%><%!
```

Comment:
With a list of properties in the object, wouldnt this cause everything to generate on one line?

Reply:
No, this is an example output:

```typescript
 export type ApiResponseError = {'detail'?: string
 'source'?: string
 'stack'?: string
 'status'?: string
 'title'?: string
 }
 ;
```

Still not perfect, but better than before (where a newline was put between each object property), even after prettification (which retained these empty lines)

Reply:
Cool!
Not perfect, but definitely better than before. Running prettier on the generated files will fix the remaining issues as well

Reply:
I improved your proposal a little further (see new commits), we now get this for objects:

```typescript
export type ApiResponseError = {
    'detail'?: string
    'source'?: string
    'stack'?: string
    'status'?: string
    'title'?: string
}
;
```

Because lack of control over Mustache's whitespace formatting (e.g. inclusion of sub templates *forces* trailing newlines) we shouldn't elaborate this further ;)